File: DSA/Sorting/C/selection.c
#include<stdio.h>
void selection(int arr[],int num){
    for (int i = 0; i < num; i++)
    {
        int min=i;
        for (int j = i+1; j < num; j++)
        {
            if(arr[min]>arr[j]){
                min=j;
            }
        }
        if(min!=i){
            int temp=arr[min];
            arr[min]=arr[i];
            arr[i]=temp;
        }
    }
    for (int i = 0; i < num; i++)
    {
        printf("%d ",arr[i]);
    }
    
    
}
int main(){
    int num;
    printf("Enter the Length: ");
    scanf("%d",&num);
    int arr[num];
    for (int i = 0; i < num; i++)
    {
        scanf("%d",&arr[i]);
    }
    
    selection(arr,num);

    return 0;
}

File: DSA/Sorting/Python/selection.py
def selection(arr):
    n=len(arr)
    for i in range(n):
        min=i
        for j in range(i+1,n):
            if arr[min]>arr[j]:
                min=j
        if(i!=min):
            temp=arr[min]
            arr[min]=arr[i]
            arr[i]=temp    
        
    return arr            
            
if __name__=="__main__":
    inp=list(map(int,input("Enter the Elements : ").split()))
    ans=selection(inp)
    print(ans)